src/helpers/communications.js

```diff
@@ -105,7 +175,11 @@ exports.createChatRoom = async (recipientUserId, initiatorUserId, initialMessage
 			if (!user.meta || !user.meta.communications_user_id) {
 				// Decrypt email and create user in communication service if `communications_user_id` is missing
 				user.email = await emailEncryption.decrypt(user.email)
-				await this.create(user.user_id, user.name, user.email, 'https://picsum.photos/200/200')
+				let userImage
+				if (user?.image) {
+					userImage = await utils.getDownloadableUrl(user.image)
```

Comment:
@nevil-mathew image should download from user service right ?

Reply:
We have added images to the user extension table right, so we will use that image path to get a downloadable URL, which will be used in communication service to download the image and add it to rocket chat.